backend/src/common/util/scheduled_workflow.go

```diff
@@ -102,11 +105,23 @@ func (s *ScheduledWorkflow) ConditionSummary() string {
 }
 
 func (s *ScheduledWorkflow) ParametersAsString() (string, error) {
-	var params []swfapi.Parameter
+	var params interface{}
 	if s.ScheduledWorkflow.Spec.Workflow == nil {
-		params = make([]swfapi.Parameter, 0)
-	} else {
+		return "", nil
+	}
+	if s.IsV1() {
 		params = s.ScheduledWorkflow.Spec.Workflow.Parameters
+	} else {
+		paramsMap := make(map[string]*structpb.Value, 0)
```

Comment:
nit: Can we create one less variable by directly using `params = make(map[string]*structpb.Value, 0)`?

Reply:
`params` is an `interface{}`. If I assign it to an empty `map`, Golang compiler complains when I try to add elements to it later. I guess this is good enough for now.

Reply:
I see! Thank you for explaining.